src/components/Card/Card.tsx

```diff
@@ -10,23 +10,24 @@ type CardProps = {
   description?: string;
   href: string;
   id?: string;
-  image?: {
-    alt?: string;
-    src?: string;
-    // sources?: {
-    //   thumbnail_lo?: string;
-    //   thumbnail?: string;
-    //   thumbnail_hi?: string;
-    // };
-  };
+  imageAlt: string;
+  imageHeight?: string;
+  imageSrc: string;
+  imageSrcSet?: string;
+  imageWidth?: string;
   meta?: {
     date?: string;
     lastUpdated?: string;
     hasType?: boolean;
     type?: string | null;
   };
   title: string;
-  variant: 'horizontal_card' | 'link_list' | 'text_list' | 'vertical_card';
+  variant:
+    | 'image_card'
+    | 'horizontal_card'
+    | 'link_list'
+    | 'text_list'
+    | 'vertical_card';
```

Comment:
This is annoying but necessary due passing `variant` from `ListingElement`

Reply:
It is what it is 🤷 

... for now at least